### ISSUE TITLE
Question: walk_tree at the instance level

### ISSUE THREAD

**Issue by nengine** [amerine/acts_as_tree#32]
Hi, Please let me know if it possible to have walk_tree at the instance level instead of class?

Instead of
```ruby
Organization.walk_tree
```
prefer
```ruby
org = Organization.find(1234)
org.walk_tree
```

**Comment by genewoo** [comment]
I don't think that works. I tested with my testing code. I am trying to crack it and give a solution.

**Comment by felixbuenemann** [comment]
Thanks to @genewoo this feature was added on master and will be available in the 2.2.0 release of the gem.

**Comment by nengine** [comment]
Many thanks to both
